source/javascripts/directives/image_loader.coffee

```diff
@@ -0,0 +1,15 @@
+EGA.directive 'imageSuggestions', ->
```

Comment:
That's what also @pferdefleisch said. It's actually none or 1 image, because I don't wanna show an empty image tag. How would you solve it?